jgrapht-core/src/test/java/org/jgrapht/alg/spanning/AhujaOrlinSharmaCapacitatedMinimumSpanningTreeTest.java

```diff
@@ -228,10 +216,10 @@ public void testInstance3()
         labels.put(5, 1);
 
         Map<Integer, Pair<Set<Integer>, Double>> partition = new HashMap<>();
-        partition.put(0, Pair.of(new HashSet<>(Collections.singletonList(1)), 1.0));
+        partition.put(0, Pair.of(new HashSet<>(Arrays.asList(1)), 1.0));
         partition.put(1, Pair.of(new HashSet<>(Arrays.asList(2, 5)), 2.0));
-        partition.put(2, Pair.of(new HashSet<>(Collections.singletonList(3)), 1.0));
-        partition.put(3, Pair.of(new HashSet<>(Collections.singletonList(4)), 2.0));
+        partition.put(2, Pair.of(new HashSet<>(Arrays.asList(3)), 1.0));
```

Comment:
Shouldn't this be Set.of(3) instad of new HashSet<>(Arrays.asList(3))

Reply:
That was my first thought too (and what I did in the first commit). But unfortunately the Set is modify when the algorithm is executed. I wanted to revert the change with my second commit, but it seems that I reverted it wrong and not to the state it was before.
I can change it back to how it was before or I could change it to `new HashSet<>(Set.of(3))` which maybe makes it more clear that the set needs to be modifiable.

Reply:
ah right, I forgot that Set.of returns an immutable set.